AcademicHub.Common/.idea/.idea.AcademicHub.Common/.idea/indexLayout.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove Rider's cache from the PR. No need to keep IDE specific files in the repo